scenes/lessons/7th_class/tarayin_artahaytutyun/tarayin_artayahtutyun.py

```diff
@@ -0,0 +1,305 @@
+    def construct(self):
+
+# INITS        
+    # 
+    
+        paymanner = VGroup(
+            Tex('Ճանապարհ - ', '700', ' (կմ)', tex_template=ARMTEX, font_size=40),
+            Tex('Արագություն - ', '60', ' (կմ/ժ)', tex_template=ARMTEX, font_size=40),
+        )
+
+        dashed_line = DashedLine().match_width(paymanner)
+
+        pahanj = Tex('Որքա՞ն ճանապարհ կմնա ', '4', ' (ժ) անց', tex_template=ARMTEX, font_size=40)
+        pahanj.next_to(dashed_line, DOWN)
+
+        line_hor = Line().match_width(pahanj).scale(1.1)
+
+        VGroup(*paymanner, dashed_line, pahanj, line_hor).arrange(DOWN, aligned_edge=LEFT).to_edge(UL)
+
+        line_hor.shift(0.3 * LEFT)
+        line_ver = Line(line_hor.get_right(), line_hor.get_right() + 3 * UP)
+
+        road = Line(stroke_width=8).scale(2.5).to_edge(RIGHT).shift([-0.6, 2, 0])
+        road_length = road.get_right()[0] - road.get_left()[0]
+        road_endpoint_left = Dot(road.get_left())
+        road_endpoint_right = Dot(road.get_right())
+
+        brace_total = Brace(road, DOWN)
+        total_distance = Tex('700', tex_template=ARMTEX).next_to(brace_total, DOWN)
+
+        car = SVGMobject('../../../../objects/SVG_files/small_car').set_color(RED).scale(0.2).move_to(road_endpoint_left)
+        car_trace = always_redraw(lambda: Line(road_endpoint_left.get_center(), car.get_center(), color=GREEN, stroke_width=8))
+
+        speed_vector = VGroup(Line().scale(0.25).rotate(PI/2).next_to(car, UP))
+        speed_vector.add(Line().scale(0.25).next_to(speed_vector[0], UP, buff=-0.02, aligned_edge=LEFT))
+        speed_vector.add(Arrow().tip.scale(0.5).next_to(speed_vector[1], RIGHT, buff=0))
+        speed_vector.add(Tex('60', font_size=35).next_to(speed_vector[1], UP))
+        
+        brace_passed = BraceBetweenPoints(road_endpoint_left.get_center(), road_endpoint_left.get_center() + 1.72 * RIGHT, UP)
+        passed_distance = Tex('240', tex_template=ARMTEX).next_to(brace_passed, UP)
+
+        brace_remaining = BraceBetweenPoints(road_endpoint_left.get_center() + 1.72 * RIGHT, road_endpoint_right.get_center(), UP)
+        harcakan = Tex('?').next_to(brace_remaining, UP)
+        remaining_distance = Tex('460', tex_template=ARMTEX).next_to(brace_remaining, UP)
+
+    # լուծում
+        solution_first_step = Tex('1) ', '60', ' $\cdot$ ', '4', ' = ', '240', tex_template=ARMTEX)
+        solution_first_step.to_edge(LEFT).shift(DOWN)
+
+        solution_second_step = Tex('2) ', '700', ' - ', '240', ' = ', '460', tex_template=ARMTEX)
+        solution_second_step.to_edge(LEFT).shift(2 * DOWN)
+    
+    # լուծումն արտահայտությամբ ժամանակի տարբեր արժեքների դեպքում
+        solution_4 = Tex('700', ' - ', '60', ' $\cdot$ ', '4', ' = ', '460')
+        solution_4.next_to(VGroup(solution_first_step, solution_second_step), RIGHT).shift(2.5 * RIGHT)
+
+        solution_5_final = Tex('700', ' - ', '60', ' $\cdot$ ', '5', ' = ', '400')
+        solution_5_final.move_to([0, -0.75, 0])
```

Comment:
Can't you use next to instead?

Reply:
կարայի, ճիշտ ես, սենց հետո դզելը ավելի բարդ ա լինելու